### ISSUE TITLE
Crash with Bundle(path: (assetPath as NSString).appendingPathComponent("DKImagePickerController.bundle"))!

### ISSUE THREAD

**Issue by VictorShurapov** [zhangao0086/DKImagePickerController#545]
Could you please help me with solution to this crash? :-)
<img width="733" alt="screen shot 2018-11-15 at 17 00 46" src="https://user-images.githubusercontent.com/12258873/48561227-2c71f780-e8f8-11e8-88b1-1c1cebcb9b91.png">

Bundle(path: (assetPath as NSString).appendingPathComponent("DKImagePickerController.bundle"))!

**Comment by zhangao0086** [comment]
Hi @VictorShurapov Could you make a mini project that can reproduce this problem?

**Comment by immustafa** [comment]
I am getting this error too, only doing initialization and presentation like you write and. is there any solution about this.  (DKImagePickerControllerResource:103)

**Comment by zhangao0086** [comment]
@immustafa Are you using Carthage or Cocoapods?

**Comment by immustafa** [comment]
yes I am using cocoapods and after install I have to upgrade to swift 4.2

**Comment by zhangao0086** [comment]
@immustafa Could you update this picker to `4.1.0` and try again?

**Comment by zhangao0086** [comment]
@immustafa Thanks! The README has been updated! 

---
#### For Swift 4.1
```
pod 'DKImagePickerController', :git => 'https://github.com/zhangao0086/DKImagePickerController.git', :branch => 'Swift4'
```

**Comment by hajiko-mobile** [comment]
I am experiencing the same issue.

I am creating a sample app that will call the DKImagePickerController like so:
```
let pickerController = DKImagePickerController()
pickerController.assetType = .allPhotos
self.present(pickerController, animated: true) {}
```

The error is the same:
![image](https://user-images.githubusercontent.com/44158116/53160406-de5d4600-3602-11e9-8853-24400a3f47cb.png)

The dependencies within the sample app are added by copying the folder content from

- Pods -> DKCamera
- Pods -> DKImagePickerController -> Sources -> DKImageImageDataManager
- Pods -> DKImagePickerController -> Sources -> DKImageFinderController

It compiles fine.

Deployment target is: 10.0
Swift: 4.2
Release used: https://github.com/zhangao0086/DKImagePickerController/releases/tag/4.1.2

Please advice, thanks!

**Comment by hajiko-mobile** [comment]
I found the reason for the crash: The DKImagePickerController.bundle was not existing which was the reason for the nil value. I had to create it within the DKImagePicker Project by selecting the DKImagePickerController.bundle target and then I copied the created bundle into my own sample project.

**Comment by zhangao0086** [comment]
@hajiko-mobile Thanks for your nice information! Unfortunately, I'm still can't reproduce it :(
I've just made an empty project that contains a Podfile like so:
```ruby
platform :ios, '10.0'
use_frameworks!
inhibit_all_warnings!

source 'https://github.com/CocoaPods/Specs.git'

project 'TestDKImagePickerController.xcodeproj'

target :TestDKImagePickerController do

    pod 'DKImagePickerController', '4.1.2'

end
```

And in the `ViewController.swift`:
```swift
override func viewDidLoad() {
    super.viewDidLoad()

    DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
        let pickerController = DKImagePickerController()
        pickerController.assetType = .allPhotos
        self.present(pickerController, animated: true) {}
    }
}
```
It works fine when I run it on the simulator. Maybe something has been ignored by me. 
Thank you any way!

**Comment by erinsnow** [comment]
> @hajiko-mobile Thanks for your nice information! Unfortunately, I'm still can't reproduce it :(
> I've just made an empty project that contains a Podfile like so:
> 
> ```ruby
> platform :ios, '10.0'
> use_frameworks!
> inhibit_all_warnings!
> 
> source 'https://github.com/CocoaPods/Specs.git'
> 
> project 'TestDKImagePickerController.xcodeproj'
> 
> target :TestDKImagePickerController do
> 
>     pod 'DKImagePickerController', '4.1.2'
> 
> end
> ```
> 
> And in the `ViewController.swift`:
> 
> ```swift
> override func viewDidLoad() {
>     super.viewDidLoad()
> 
>     DispatchQueue.main.asyncAfter(deadline: .now() + 2) {
>         let pickerController = DKImagePickerController()
>         pickerController.assetType = .allPhotos
>         self.present(pickerController, animated: true) {}
>     }
> }
> ```
> 
> It works fine when I run it on the simulator. Maybe something has been ignored by me.
> Thank you any way!

因为你测试的不是你 Readme 里告诉我们的分支。
pod 'DKImagePickerController', :git => 'https://github.com/zhangao0086/DKImagePickerController.git', :branch => 'Swift4'
请不要自由发挥成 pod 'DKImagePickerController', '4.1.2'
Readme 里给我们用的这个分支里没有 bundle，里面空空如也。

**Comment by jemzubiri** [comment]
> I found the reason for the crash: The DKImagePickerController.bundle was not existing which was the reason for the nil value. I had to create it within the DKImagePicker Project by selecting the DKImagePickerController.bundle target and then I copied the created bundle into my own sample project.

How exactly did you do this? @hajiko-mobile

**Comment by Erzen-sys** [comment]
> > I found the reason for the crash: The DKImagePickerController.bundle was not existing which was the reason for the nil value. I had to create it within the DKImagePicker Project by selecting the DKImagePickerController.bundle target and then I copied the created bundle into my own sample project.
> 
> How exactly did you do this? @hajiko-mobile

>It works for me
